t/14-isotovideo.t

```diff
@@ -47,4 +47,16 @@ EOV
     is_in_log('\d* scheduling.*shutdown', 'schedule has been evaluated');
 };
 
+subtest 'isotovideo with custom git repo parameters specified' => sub {
+    chdir($pool_dir);
+    unlink('vars.json');
+    mkdir('repo.git');
+    qx{git init -q --bare repo.git};
+    remove_tree('repo');
```

Comment:
Where did "repo" come from?

Reply:
I added a comment in file. This is to ensure the checkout folder does not exist so that git clone tries to create a new checkout on every test run.